en/02_Developer_Guides/08_Performance/08_Other_Optimisations.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Other Optimisations
+summary: Other performance optimisations
+---
+
+# Other optimisations
+
+Here are some other backend performance optimisations that can be made.
+
+## Increasing available resources
+
+Silverstripe CMS tries to keep its resource usage within the documented limits
+(see the [server requirements](../../getting_started/server_requirements)).
+
+These limits are defined through `memory_limit` and `max_execution_time` in the PHP configuration. These can be updated with [`Environment::increaseMemoryLimitTo()`](api:SilverStripe\Core\Environment::increaseMemoryLimitTo()) which will call `ini_set('memory_limit', <limit>)`, and [`Environment::increaseTimeLimitTo()`](api:SilverStripe\Core\Environment::increaseTimeLimitTo()) which will call `set_time_limit(<limit>)`
+
+> [!NOTE]
+> Most shared hosting providers will have maximum values that can't be altered.
+
+For certain tasks like synchronizing a large `assets/` folder with all file and folder entries in the database, more
+resources are required temporarily. In general, we recommend running resource intensive tasks through the
+[command line](../cli), where configuration defaults for these settings are higher or even unlimited.
+
+```php
+// app/src/Services/MyClass.php
+namespace App\Services;
+
+use SilverStripe\Core\Environment;
+
+class MyClass
+{
+    public function increaseLimits(): string
+    {
+        Environment::increaseMemoryLimitTo('256M');
+        Environment::increaseTimeLimitTo(500);
+    }
+}
+```
+
+## Cache warming
+
+For sites employing caching, cache warming involves proactively populating the cache with frequently accessed data. This reduces cache misses and ensures rapid response speeds from the outset, providing users with up-to-date content efficiently.
+
+Here are some strategies for cache warming:
+
+- Generate cache entries for the most popular pages or data on deployment, for instance by implementing the [`Flushable`](api:SilverStripe\Core\Flushable) on a class that then pre-fetches data.
```

Comment:
`Flushable` is for flushing... I'm not sure what it has to do with pre-warming cache?

Reply:
Removed the mention of `Flushable`